conans/client/file_copier.py

```diff
@@ -145,6 +145,8 @@ def _filter_files(src, pattern, links, excludes, ignore_case, excluded_folders):
             relative_path = os.path.relpath(root, src)
             compare_relative_path = relative_path.lower() if ignore_case else relative_path
             for exclude in excludes:
+                if ignore_case:
+                    exclude = exclude.lower()
```

Comment:
at line 121 it does the same.

Reply:
good point. But need to remove the 2 lines.